cmd/registry/cmd/config/configurations/activate.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configurations
+
+import (
+	"fmt"
+
+	"github.com/apigee/registry/pkg/connection"
+	"github.com/spf13/cobra"
+)
+
+func activateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "activate CONFIGURATION_NAME",
+		Short: "Activates an existing named configuration.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			name := args[0]
+			if err := connection.ValidateConfigName(name); err != nil {
+				return err
+			}
+
+			_, err := connection.ReadConfig(name)
+			if err != nil {
+				return fmt.Errorf("Cannot read config %q: %v", name, err)
+			}
+
+			err = connection.ActivateConfig(name)
+			if err != nil {
+				return fmt.Errorf("Cannot activate config %q: %v", name, err)
+			}
+
+			fmt.Printf("Activated %q.\n", name)
```

Comment:
Similar to error handling, we usually use `log.Infof` to print information for users.

Reply:
discussion above